11-query/03-insert-query.md

```diff
@@ -80,6 +80,55 @@ This `match insert` query:
 3. Inserts an `employment` relation with `$org` and `$person` as its roleplayers, assigned to variable `$new-employment`.
 4. Inserts the ownership of `reference-id` with value `WGFTSH` to the `$new-employment` relation instance.
 
+### Duplicate Role Players
+
+As of Grakn 1.7.0, relations are allowed to have duplicate role players. This means the following would be valid in Grakn:
```

Comment:
- so far, we have no other instance of mentioning the version, when documenting a feature. if you think it's important to note the version here, do you think a _Colored Panel_ (documented in the readme) would be more appropriate?

- we seem to be using `roleplayers` instead of `role players`

Reply:
agreed will use Colored Panel
I will change it to role players because that is more correct...